web/client-api/src/main/java/io/deephaven/web/client/api/Sort.java

```diff
@@ -7,7 +7,13 @@
 import jsinterop.annotations.JsMethod;
 import jsinterop.annotations.JsProperty;
 
+/**
+ * Describes a Sort present on the table. No visible constructor, created through the use of Column.sort(), will be tied
+ * to that particular column data. Sort instances are immutable, and use a builder pattern to make modifications. All
+ * methods return a new Sort instance.
+ */
 public class Sort {
+
     @JsProperty(namespace = "dh.Sort")
     public static final String ASCENDING = "ASC",
```

Comment:
note to future me, make this into a tstypedef since only three strings are legal anyway...